admin/client/utils/queryParams.js

```diff
@@ -45,3 +45,11 @@ export function stringifyColumns (columns, defaultColumnPaths) {
 	if (columnString === defaultColumnPaths) columnString = undefined;
 	return columnString;
 }
+
+export function parametizeFilters (filterArray) {
```

Comment:
Yes, which implies we should move the function that reads these back to be loaded in to state should also be testable, since currently we need the two to work in harmony more than we need them to appear a specific way.